setup.cfg

```diff
@@ -39,16 +45,18 @@ exclude =
 
 [options.extras_require]
 ws =
-    websockets==9.*
+    websockets~=10.2
```

Comment:
I think websockets `10.x` drops support for Python 3.6, this might have to stay on `9.x` for now

Reply:
maybe it's okay to drop 3.6 support as well for pygls? Would be cool to see this PR get forward.

Reply:
Perhaps, though I think that call probably lies with @danixeee  or @dgreisen

Reply:
Is it necessary to update `websockets` to support python 3.10? 


If not, I'd propose doing it in a separate PR, while merging this first, to address compatibility issues. 

Then we can reason about whether dropping python3.6 is a good idea.